scripts/test-truncating-of-client-hello.py

```diff
@@ -188,7 +188,7 @@ def run_all(conversations):
         try:
             runner.run()
         except:
-            # TBD: print(traceback.format_exc())
+            print(traceback.format_exc())
```

Comment:
that is good idea I have implemented part of this behavior. Would you like have logging thru this https://docs.python.org/2/library/logging.html? I have noticed that in some part of the code are also TBD with logging mentioned (like in runner code).